CMakeLists.txt

```
@@ -16,6 +17,14 @@ include(GNUInstallDirs)
 # Effectively adds '-fpic' flag to CXX_FLAGS. Needed for dynamic catalogues.
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+# Have LTO where possible, ie add -flto
+check_ipo_supported(RESULT HAVE_LTO OUTPUT ERR_LTO)
+if(HAVE_LTO)
```

```suggestion
if(HAVE_LTO AND NOT DEFINED CMAKE_INTERPROCEDURAL_OPTIMIZATION)
```
It could be useful to still be able to override this. Setting it as a cache variable would affect any other project that includes arbor, so just checking if it is already set might be a better solution. Alternatively there there could be an `ARB_LTO` option.

---

Good idea!